frappe_pwa/www/pwa.js

```diff
@@ -52,7 +64,9 @@ if ('serviceWorker' in navigator) {
     window.addEventListener('beforeinstallprompt', function (e) {
         e.preventDefault();
         installPromptEvent = e;
-
+        if (window.location.pathname === '/install') {
+            show_prompt_with_installation();
+        }
```

Comment:
@AminovE99 Mentioned before, but the code to display the "_Install PWA_" notification should be in the `showAddToHomeScreen` function...

Also, the idea is to display the install button on any page, not just the install page! How is the user supposed to find the install page otherwise ??

The expected behavior is to display a popup to invite the user to got to the install page, where he can find the install button and instructions on how to install.

Reply:
@madmath03 I fix this notes in new commits